build/maven.go

```diff
@@ -45,7 +45,8 @@ type MavenModule struct {
 	// A pipe to write the maven extractor output to.
 	outputWriter io.Writer
 	// Path to the build info temp file that will be generated by the maven extractor.
-	buildInfoPath string
+	buildInfoPath  string
+	rootProjectDir string
```

Comment:
Since this is the only struct property that doesn't have a description and to maintain consistency and symmetry, let's add a description for this property.